ニュースサイトの作成

h-haraguchi opened the pull request:
まだできてないこと（今回の対応対象外。）

- [x] タイトル上部の日付とタグのスタイリング
- [x] コメント付きの画像をもっと合間に入れる予定
- [x] フッタ
- [x] ヘッダのハンバーガメニュー
- [x] MNR社のリンク付与

h-haraguchi committed:
初期実装

h-haraguchi committed:
firebaseを導入

h-haraguchi committed:
add readme for install

h-haraguchi committed:
version up node and add script to package.json

h-haraguchi committed:
fix firebase command

h-haraguchi committed:
思想と今後の展望を追加

firebase用jsは特に今回使用しないため削除
元ファイルが残っていたため削除
テキスト追加に伴う画像とスタイルの追加・変更

h-haraguchi committed:
add start and deploy manual to readme

tronperidot commented on web/public/index.html:
```diff
@@ -0,0 +1,212 @@
+        <p>
+          <span class="character blue">原口さん：</span>
+          デリダじゃないですが、引き続き「脱効率」を図り、本質に理解していただくための機会を提供する製品を作っていこうと思います。
+          デリだといえば、MNRが行おうとしているのは差延へのまなざし自体に対する反省活動と言えるかもしれません。
```
```suggestion
          デリダといえば、MNRが行おうとしているのは差延へのまなざし自体に対する反省活動と言えるかもしれません。
```

tronperidot commented on web/public/index.html:
```diff
@@ -0,0 +1,212 @@
+          そして気づいたのです。<strong>"彼ら"は動いていなかった</strong>のです。
+          そもそも、アシスト(assist)の語源は「そばに(as)立つ(sist)」です。
+          しかし彼らは私がどこに行ったとしても、その場から動かないのです。このようなアシスタントがありえるでしょうか。いや、ありえない。
+          私は本当のアシスタントとは何かというのを知らしめたくなりました。本当の、"動く"AIアシスタントの誕生です。
```
動くアシスタントじゃなくて、"そばに立つ"アシスタントの方が個人的には好みだけどどうかな

h-haraguchi replied:
対応した！確認よろ

tronperidot commented:
全般的に「アウフヘーベン」とか「デルタ」とかは、一般が理解できない気がする（というか俺が調べないと分からなかった。）ので、wikipediaのように最後か途中かで注釈を入れた方がいい気がしている。

takashi1coq commented:
インタビュー記事によくある、みみずがどんなライターか説明入れたほうがいいと思う

tronperidot committed:
誤字修正

デリだ→デリダ

Co-Authored-By: h-haraguchi <account.cgtdnpck@me.com>

h-haraguchi committed:
add style to timestamp and tag

h-haraguchi committed:
modify literal from 動く to そばに立つ

see:
https://github.com/tamakon/drone-alexa/pull/2#discussion_r257563943

h-haraguchi committed:
make footer

h-haraguchi committed:
add header menu and translation

h-haraguchi committed:
add author info and related articles

h-haraguchi commented:
> インタビュー記事によくある、みみずがどんなライターか説明入れたほうがいいと思う

ざっくりとだけ入れたで。残りの細かい説明はあとでみんなで考えよう！

h-haraguchi committed:
aggregate scripts

h-haraguchi committed:
add english version

h-haraguchi committed:
fix typos

h-haraguchi committed:
add favicon

tronperidot commented on web/public/index.html:
```diff
@@ -0,0 +1,394 @@
+        <p>
+          <span class="character green">みみず：</span>
+          なるほど。動くのが本当のAIアシスタントだ！という思いから本製品が作られたのですね。
+          私は売れるもの、役に立つものであれば、どのような形でもいいと思ってしまうのですが、実際に動かないことでなにかデメリットがあったのでしょうか？
```
> 私は売れるもの、役に立つものであれば、どのような形でもいいと思ってしまう

この表現は直接的すぎてインタビューとして不適切な気がする。
この質問ではこの製品に本当に価値があるのかと問いたいのだから、
「すでにあるsiriやgoogle homeで十分のように思えるが、動くことによるメリットはあるのか」のような質問になる気がする

h-haraguchi replied:
対応した！

tronperidot commented on web/public/index.html:
```diff
@@ -0,0 +1,394 @@
+          結果的には効率を原因とする格差があり、目に見えるものだけが優先されます。
+          目に見えることはわかりやすく、わかりやすいことは理解の効率に繋がるからです。
+          皆さんは物事を理解しようとします。なぜなら知っていた方が今後の人生において効率が良いからです。
+          効率、効率、全て効率です。キリスト教のような一神教を批判する方々も効率という一神を信仰しているようです。
```
> キリスト教のような一神教を批判する方々も効率という一神を信仰しているようです。

皮肉入れる必要ある？w
「これはまるでキリスト教や仏教のように、効率を信仰しているように思えます。」とかその程度に抑えていいんじゃない？w

h-haraguchi replied:
信仰そのものが悪いと言っているわけではなく、一つの考え方だけを信仰する様をおかしいと思っている大衆のほとんどが唯一神のごとく効率を扱っている様を批判しているので、そのフレーズにはできないかな。
変えるとしたら、「これはまるでキリスト教やイスラム教のように、効率という唯一神を信仰しているように思えます。」でどうすか？
仏教は唯一神の宗教じゃないからイスラム教に変えた。

tronperidot replied:
なるほどね。そういうことなら、その提案の文ならいい感じかな。

h-haraguchi replied:
対応した！

tronperidot commented on web/public/index.html:
```diff
@@ -0,0 +1,394 @@
+          それぞれの主義における理想の欠陥を受け入れ、それぞれの主義を共存している主義と受け入れることで論理的な解決が行われます。
+          では実践的、具体的な解決としてはどうなるかというと、それはどちらかの主義における理想を諦めることではないのです。
+          そもそもそれらは共存しているので諦めるという行為自体がなくなるのです。つまりは「共存している」ということを理解すればいいのです。
+          いろいろ言いましたが、要は言葉の過ちを認めるということです。
```
いろいろ言いましたが」の前にみみずのツッコミを入れたいと個人的には思っている
みみずに「えっとつまりどういうことですか？」
とか入れて読者を落ち着かせないとこれ情報量多すぎる気がしているw
せっかくのまとめが処理に追いつかなくて読み捨てられそう。

h-haraguchi replied:
対応した！

tronperidot reviewed (commented):
今気になっている点ね。
議論にするか直すかは任せます。
来週集まった時に詰めるでもいいかなあとは思っている

h-haraguchi committed:
fix phrase question of drone-alexa merit

review point: https://github.com/tamakon/drone-alexa/pull/2#discussion_r261852419

h-haraguchi committed:
make rhythm among long sentence

review point: https://github.com/tamakon/drone-alexa/pull/2#discussion_r261852561

h-haraguchi committed:
exclude sarcasm from sole god phrase

tronperidot reviewed (approved):
いい感じ！一旦はこれでいいかな！

takashi1coq commented:
読んだ。とりあえずマージでよろしいのでは

h-haraguchi committed:
add interview photos

h-haraguchi committed:
fix product name and update youtube url

h-haraguchi committed:
add google analytics

h-haraguchi committed:
reduce picture size

h-haraguchi committed:
update english page

h-haraguchi committed:
add yamashita's joke

h-haraguchi committed:
fix mistake word
